blocklylib-core/src/main/res/values/attrs.xml

```diff
@@ -30,7 +30,13 @@
     </declare-styleable>
 
     <declare-styleable name="WorkspaceFragment">
```

Comment:
Since you're including buttons, this should probably be pushed into the BlocklyWorkspaceTheme.  The WorkspaceFragment doesn't include the zoom buttons.  If it is part of the theme, then the Activity and WorkspaceView can each get the respective values without one needing a reference to the other.